bob-access/src/credentials.rs

```diff
@@ -11,7 +11,8 @@ use crate::{error::Error, extractor::ExtractorExt, Authenticator};
 
 #[derive(Debug, Default, Clone)]
 pub struct Credentials {
```

Comment:
This struct solving 2 separate tasks: 
1. It stores Credentials (`CredentialsKind`) and associated info (`hostname` and `Vec<SocketAddr>`) from config to compare with
2. It stores Credentials (`CredentialsKind`) and associated info (`SocketAddr`) received from request that compares with the first one

Previously, the stored data was the same, and we used the same struct for 2 tasks. Currently, the stored data are different, so the struct should also be split into 2 different ones.

The first one can be named `DeclaredCredentials`, the second one - `RequestCredentials`